hw_2.js

```diff
@@ -0,0 +1,118 @@
+"use strict";
+
+// // Task 1
+
+// let x = 20;
+// let y = 58;
+// let z = 42;
+
+// console.log(x, y, z);
+
+// // Task 2
+
+// let seconds = 60;
+// let minutes = 60;
+// let hours = 24;
+// let days = 365;
+// let myAgeInSeconds = seconds * minutes * hours * days;
+
+// console.log(myAgeInSeconds);
+
+// Task 3
+
+// let count = 42;
+// let userName = '42';
+
+// count = String(count); // Метод 1
+// console.log(count);
+// console.log(typeof count);
+
+// count.toString; // Метод 2
+// console.log(count);
+// console.log(typeof count);
+
+// userName = Number(userName); // Метод 1
+// console.log(userName);
+// console.log(typeof userName);
+
+// userName * 1; // Метод 2
+// console.log(userName);
+// console.log(typeof userName);
+
+// Task 4
+
+// let a = 1;
+// let b = 2;
+// let c = "белых медведей";
+
+// a = "1";
+// b = "2 ";
+
+// let twelveBears = a + b + c;
+// console.log(twelveBears);
+
+// Task 5
+
+// let word1 = "доступ";
+// let word2 = "морпех";
+// let word3 = "наледь";
+// let word4 = "попрек";
+// let word5 = "рубило";
+
+// let lengthWords = word1.length+word2.length+word3.length+word4.length+word5.length;
```

Comment:
между операндами и операторами принято ставить пробелы
```suggestion
// let lengthWords = word1.length + word2.length + word3.length + word4.length + word5.length;
```